app/query_objects/filtered_tasks_query.rb

```diff
@@ -8,4 +8,12 @@ def by_status(relation, status)
   def by_title(relation, title)
     relation.where("title LIKE ?", title)
   end
+
+  def by_description(relation, description)
+    relation.where("description LIKE ?", description)
+  end
+
+  def by_deadline_at(relation, deadline)
+    relation.where(deadline_at: deadline)
```

Comment:
It has no sense, since it must be equal even by seconds

Reply:
It will be better if I implement it as follows:
```suggestion
    relation.where(deadline_at: deadline.midnight..(deadline.midnight + 1.day))
```
?

Reply:
Do you even use this method somewhere?